### DISCUSSION TITLE
[Question]: Can't chat with picture

### DISCUSSION THREAD

**Discussion by JamieWuJinYi** [danny-avila/LibreChat#2747] [Troubleshooting] [answered]
### What is your question?

I can‘t chat with picture
Chat with files is ok.

Are there any special settings?

### More Details

"version": "0.7.0",

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/100826652/78ed7207-edab-4bc6-a687-09eda7946e8c)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
It should be working with your selected model.

 For Azure/OpenAI, this is any model including “gpt-4o”, ”gpt-4-turbo”, or “gpt-4-vision-preview”

what is the error you’re receiving?

**Reply by danny-avila** [reply, marked as answer]
Also I recommend updating to the latest version and not using 0.7.0 for expected behavior

**Reply by JamieWuJinYi** [reply]
After upgrading to the latest version, we can now chat with pictures. Thank you very much.
